integration-tests/sdk/channels.it.js

```diff
@@ -178,6 +164,27 @@ describe('Channels', () => {
       expect(response).toHaveLength(resultCount)
     })
   })
+
+  it('delete', () => {
+    const uri = service
+      .byId(channelResponse.id)
+      .withVersion(channelResponse.version)
+      .build()
+
+    const deleteRequest = {
+      uri,
+      method: 'DELETE',
```

Comment:
Don't we expose request methods as constants somewhere?
any way we can eliminate hard coded values?